_maps/templates/lavaland_surface_golem_ship.dmm

```diff
@@ -36,7 +36,7 @@ aaaaaaaaaaaaaaaaaaaa
 aaaaabbbbbbbbbbbbbbb
 aaaaabccccccdddefegh
 aaaaabffffffffiefegh
-aabbbbjkbbbbbbbbbbbb
+aabbbbkkbbbbbbbbbbbb
```

Comment:
This is actually the only line that matters. All the rest is formatting issues made by the map merger.